app/models/employee.rb

```diff
@@ -5,4 +5,7 @@ class Employee < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :given_kudo, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :kudo
+  has_many :receiven_kudo, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :kudo
```

Comment:
- after has_many there should be plural name, so it should be :given_kudos, and :received_kudos 
- inverse_of values are incorrect, but on the other hand - do we really need them here?

Reply:
I corrected it and removed inverse_of, it is not needed in this case